main/src/cache.rs

```diff
@@ -183,7 +183,8 @@ pub async fn place_bid(cfg: &CacheBidConfig) -> Result<()> {
 
     greyln!("Checking if contract can be cached...");
 
-    let raw_output = place_bid_call.clone().call().await;
+    let from_address = wallet.default_signer().address();
```

Comment:
You could move this line to 170-171 and avoid cloning the wallet.

Reply:
Cool, thanks!